src/constants/buildings-custom-info.constant.vue

```diff
@@ -294,9 +294,21 @@ export const BuildingsCustomInfo: { [buildingId: string]: IBuildingCustomInfo }
   '101890': {owner: BuildingOwners.northwestern.key},
   // Wieboldt Hall, 340 E. Superior
   '256407': {owner: BuildingOwners.northwestern.key},
-  // Feinberg Pavilion
+ // Chicago Campus 2, Feinberg Pavilion, 251 E Huron St
   '103632': {owner: BuildingOwners.northwestern.key},
 
+  // Prentice Women's Hospital, 250 E Superior St
+  '103615': {owner: BuildingOwners.northwestern.key},
+  // NMH Arkes Family Pavilion, 676 N St Clair St
+  '103684': {owner: BuildingOwners.northwestern.key},
+  // 8792, Feinberg School of Medicine, 303 E Chicago Ave
+  '256411': {owner: BuildingOwners.northwestern.key}, 
+  // 645 North Michigan
```

Comment:
I'm not sure this one is owned by Northwestern, actually, they might just rent space: https://645nmichigan.com/